### DISCUSSION TITLE
Prisma generated query for mongodb

### DISCUSSION THREAD

**Discussion by jagadeeshmusali** [prisma/prisma#24668] [Q&A]
### Question

I am trying to log generated query. Almost all of the times am noticing that its generating aggregation query instead of a regular mongo query. Even for a simple `find`
Is it always the case? or does it also generate a regular mongo query instead of aggregation query?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @jagadeeshmusali 👋 

Prisma Client does generate aggregation queries for many operations, including simple finds. This is likely due to the way it translates its higher-level query language into MongoDB queries. If you are experiencing performance issues, you can open an [issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) or if you need more control over the queries, you can use [raw queries](https://www.prisma.io/docs/orm/prisma-client/queries/raw-database-access/raw-queries#raw-queries-with-mongodb).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
